src/i_video.c

```diff
@@ -890,6 +890,15 @@ int vga_porch_flash; // emulate VGA "porch" behaviour
 int fps; // [FG] FPS counter widget
 int widescreen; // widescreen mode
 
+static inline void I_RenderFrame (void)
```

Comment:
Not sure if this function name makes sense? 🤔

Reply:
We can name it `I_UpdateRender()`.

Reply:
Yep, that's better. Thank you!